client/keystore/src/local.rs

```diff
@@ -234,6 +236,69 @@ impl Keystore for LocalKeystore {
 		Ok(sig)
 	}
 
+	#[cfg(feature = "bandersnatch-experimental")]
+	fn bandersnatch_public_keys(&self, key_type: KeyTypeId) -> Vec<bandersnatch::Public> {
+		self.public_keys::<bandersnatch::Pair>(key_type)
+	}
+
+	/// Generate a new pair compatible with the 'bandersnatch' signature scheme.
+	///
+	/// If `[seed]` is `Some` then the key will be ephemeral and stored in memory.
```

Comment:
This seems like a very significant change in behavior triggered by something that's very subtle? Maybe it'd be better to have two methods here instead so that this is more clear? e.g.;

```rust
	#[cfg(feature = "bandersnatch-experimental")]
	fn bandersnatch_generate_new(
		&self,
		key_type: KeyTypeId,
	);

	#[cfg(feature = "bandersnatch-experimental")]
	fn bandersnatch_generate_new_ephemeral(
		&self,
		key_type: KeyTypeId,
		seed: &str,
	);
```

But that's just my personal optinion; feel free to ignore.

Reply:
Fair enough. Would be nice to refactor this in the future, in my opinion.